src/gameplay_constants.ts

```diff
@@ -2,3 +2,9 @@ import { UInt32 } from 'snarkyjs';
 
 export const MELEE_ATTACK_RANGE = 24;
 export const MELEE_ATTACK_RANGE_U32 = UInt32.from(MELEE_ATTACK_RANGE);
+
+export const ARENA_WIDTH = 800;
+export const ARENA_HEIGHT = 800;
```

Comment:
Are these correct?

Reply:
It looks like in `mina-arena` the `MinaArenaClient` has a `createGame` method which creates a game with an arena of width 650 and height 550. `mina-arena-server` requires width and height and does not have default values for them, instead requiring the client to provide them.

I would be okay with switching to 800x800, but the client is doing 650x550 currently it seems.

Reply:
That's right.  I'll stick with that for now.  I remember rendering a true square arena looked weird because most screens themselves are not square.  Our UI mentor will probably have something to say about that too.